core/src/main/java/org/jdbi/v3/tweak/CollectorFactory.java

```diff
@@ -30,7 +29,7 @@
      * @param type the object type
      * @return {@code true}, if accepts, otherwise {@code false}
      */
-    boolean accepts(Class<?> type);
+    boolean accepts(ResolvedType type);
```

Comment:
We could still use `TypeResolver` internally e.g. `TypeResolver.resolve(Type)` to get a resolved type.

At the moment though it looks like we're just getting the erased type from the resolved type and using that. :-/